src/firefly/js/tables/ui/TableRenderer.js

```diff
@@ -147,8 +155,8 @@ export class SelectableCell extends React.Component {
 /*---------------------------- CELL RENDERERS ----------------------------*/
 
 function getValue(props) {
-    const {rowIndex, data, col} = props;
-    return get(data, [rowIndex, col], 'undef');
+    const {rowIndex, data, colIdx} = props;
+    return get(data, [rowIndex, colIdx], 'undef');
```

Comment:
Does 'undef' mean return a 'undef' string or 'undefined' type in case the result of get is undefined?